perun-core/src/main/java/cz/metacentrum/perun/core/entry/UsersManagerEntry.java

```diff
@@ -693,6 +693,17 @@ public void removeUserExtSource(PerunSession sess, User user, UserExtSource user
 	}
 
 	@Override
+	public void removeUserExtSources(PerunSession sess, User user, List<UserExtSource> userExtSources, boolean forceDelete) throws UserNotExistsException, UserExtSourceNotExistsException, PrivilegeException, UserExtSourceAlreadyRemovedException {
+		for (UserExtSource src : userExtSources) {
+			if (forceDelete) {
+				removeUserExtSource(sess, user, src, true);
+			} else {
+				removeUserExtSource(sess, user, src);
+			}
```

Comment:
You can theoretically simplify it.
```suggestion
			removeUserExtSource(sess, user, src, forceDelete);
```